Test/Tests/Component/AgentRestart.Tests.ps1

```diff
@@ -112,52 +107,56 @@ Test-WithRetries 3 {
         }
 
         BeforeAll {
+            $Testenv = [TestenvConfigs]::New($TestenvConfFile)
             Initialize-PesterLogger -OutDir $LogDir
-            $MultiNode = New-MultiNodeSetup -TestenvConfFile $TestenvConfFile
+            $MultiNode = New-MultiNodeSetup `
+                -Testbeds $Testenv.Testbeds `
+                -ControllerConfig $Testenv.Controller `
```

Comment:
What about passing just $Testenv variable when using more than two fields of the class?

Reply:
The idea is not to do this :) This will only make code few lines shorter, but the usage will be obscured.